skycultures/arabic_lunar_stations/reference.fab

```diff
@@ -0,0 +1,7 @@
+1|IBN QUTAYBA, AD-DINAWARI, ABŪ MUḤAMMAD ‘ABDULLAH B. (died ~ 889 AD), KITĀBU’L-ANWĀ’ (on meteorolgy of the Arabs), (Arabic print of the original book, DAIRUT’L-MA‘ARIFI’L-OSMANIA, Hydarabad, India, 1956 AD), أبو محمد عبد الله بن مسلم ابن قتيبة الدينوري (توفي 276 هـ)، كتاب الأنواء في مواسم العرب، دائرة المعارف العثمانية، حيدر أباد، الهند، 1375 هـ|
+2|Al-Marzoqi, Abū Ali Aḥmad b. Muḥammad b. Al-Ḥassan (died 1030 AD), Al-Azminah wal’Amkinah (Times and Places), (Arabic print of the original book in 2002, World of Books, Beirut, Lebanon)., الإمام أبو علي أحمد بن محمد بن الحسن المرزوقي (توفي سنة 421 هـ)، الأزمنة والأمكنة، تحقيق د. محمد نايف الدليمي، عالم الكتب، بيروت، لبنان، 1422 هـ.|
+3|Al- Thaqafī, Abdu’Allah b. Ḥussayn b. Aṣim (died ~ 1012 AD), Al-Anwā‘ wal’Azminah (Meteorology and times), (Arabic print of the original book in 1996, Dar Al-Jeel, Beirut, Lebanon)., عبد الله بن حسين بن عاصم الثقفي (توفي 403 هـ)، الأنواء والأزمنة، تحقيق: د. نوري حمودي القيسي، د. محمد نايف الدليمي، دار الجيل، بيروت، لبنان، 1416 هـ.|
+4|Book of the Stars by Al-Sufi (died 986): Critical edition with commentary by Khalid AlAjaji, digital edition, 2021 AD, كتاب الكواكب لأبي الحسين عبد الرحمن بن عمر الرازي المعروف بالصوفي (291-376)، تحقيق خالد بن عبد الله العجاجي|https://drive.google.com/drive/folders/1s6JXzftwjMQ5rgZoGE3718EtBLBZtjzr?usp=sharing
+5|Al-farāhīdī, Abu Abdul’Raḥmān Al-ḫalīl b. Aḥmad (died ~ 786 AD), Kitab Al-Ayn (The book of Arabic letter العين), (Arabic print of the original book, 1980 – 1985)., أبو عبد الرحمن الخليل بن أحمد الفراهيدي (توفي 170 ه)، كتاب العين، 1400 – 1405 هـ.|

```

Comment:
again, capitalize author name. (also al-Halil with small al- and capital H ? I am no expert in these things, though.) If yes, please check the other names for these details.

Reply:
Prof. Paul Kunitzsch uses proper names with small al-.  I found these two links in agreement with Prof. Kunitzsch usage:
- [al- Arabic language](https://www.britannica.com/topic/al-Arabic-language)
- [The Chicago Manual of Style, sixteenth edition (2010), The Arabic definite article](https://english.stackexchange.com/questions/498568/i-am-trying-to-find-out-if-there-is-a-convention-of-correctness-for-writing-arab)
I will try to conform to this in all relevant files.

Reply:
Thank you. Yes, I have a few of Kunitzsch's works, so I got to use that style.